build.sh

```diff
@@ -5,9 +5,12 @@ cfssl genkey -initca ca.json | cfssljson -bare ca
 cfssl genkey intermediate.json | cfssljson -bare intermediate
 cfssl sign -config config.json -profile CA -ca ca.pem -ca-key ca-key.pem intermediate.csr intermediate.json | cfssljson -bare intermediate
 cfssl genkey strimzi.json | cfssljson -bare strimzi
-cfssl sign -config config.json -profile CA -ca intermediate.pem -ca-key intermediate-key.pem strimzi.csr strimzi.json | cfssljson -bare strimzi
+cfssl sign -config config.json -profile clusterCA -ca intermediate.pem -ca-key intermediate-key.pem strimzi.csr strimzi.json | cfssljson -bare strimzi
 cfssl genkey clients.json | cfssljson -bare clients
-cfssl sign -config config.json -profile CA -ca intermediate.pem -ca-key intermediate-key.pem clients.csr clients.json | cfssljson -bare clients
+cfssl sign -config config.json -profile clientsCA -ca intermediate.pem -ca-key intermediate-key.pem clients.csr clients.json | cfssljson -bare clients
```

Comment:
I don't think this is correct. The intermediate CAs need to be marked as CAs. the profiles you added do not do it.